AlphaWallet/Core/Helpers/BlockieGenerator.swift

```diff
@@ -29,6 +29,21 @@ enum BlockiesImage {
     }
 }
 
+extension BlockiesImage: Hashable {
+    static func == (lhs: BlockiesImage, rhs: BlockiesImage) -> Bool {
```

Comment:
Isn't the auto-synthesized version work the same way?

Reply:
not clearly understand, could you please explain?

Reply:
I mean: if you removed this `==` function, it would still work because I think the implementation works the same as the auto-generated version.